README.md

```diff
@@ -128,5 +128,5 @@ Right now I'm working on spoofing Joycon with a microcontroller, it's basically
 
 ## Ending remarks
 
-Right now I only took apart one left Joycon and yet to tear into the console itself, because I want to finish the Zelda first. But so far it looks like Nintendo make some really weird design decisions both in software and in hardware, probably to make what I'm doing more difficult. Anyway, I'll update this from time to time when I have new discoveries. Please share if you find this useful.
+Right now I have only took apart one left Joycon and have yet to tear into the console itself, because I want to finish the Zelda first. But so far it looks like Nintendo has made some really weird design decisions both in software and in hardware, will probably make what I'm doing more difficult. Anyway, I'll update this from time to time when I have new discoveries. Please share if you find this useful.
```

Comment:
Comma splices are fine in informal language, but it's kinda the antithesis of a pedantic grammar PR